index.js

```diff
@@ -6,6 +6,8 @@ var os = require('os');
 var path = require('path');
 var resolve = require('resolve').sync;
 var spawn = require('child_process').spawn;
+var glob = require("glob");
+var async = require("async");
```

Comment:
Well the operation itself (spawning a child process and waiting for it to end) is still async so a simple for loop wouldn't work. 

I could replicate `async`'s series functionality with a well crafted recursive function, but `async` does it better and is more well tested so it seemed a better fit for this wrapper.